backend/lib/pix-service/service.js

```diff
@@ -0,0 +1,41 @@
+import { Value } from '@sinclair/typebox/value'
+import { PIECE_INDEXER_URL } from '../config.js'
+import { getMinerPeerId } from '../rpc-service/service.js'
+import { PixResponse } from './data-types.js'
+import pRetry from 'p-retry'
+
+/**
+ * @param {string} providerId
+ * @param {string} pieceCid
+ * @returns {Promise<string>}
+ */
+export const pixRequest = async (providerId, pieceCid) => {
```

Comment:
Maybe naming this function something like `getDealPayloadCid` would make more sense as `pixRequest` is too generic, what do you think?